worlds/v6/__init__.py

```diff
@@ -0,0 +1,53 @@
+import string
+from .Items import item_table, V6Item
+from .Locations import location_table, V6Location
+from .Options import v6_options
+from ..generic.Rules import set_rule
+from BaseClasses import Region, RegionType, Entrance, Item, MultiWorld
+from ..AutoWorld import World
+
+client_version = 1
+
+
+class V6World(World):
+    """ 
+     VVVVVV is a platform game all about exploring one simple mechanical idea - what if you reversed gravity instead of jumping?
+    """ #Lifted from Store Page
+
+    game: str = "VVVVVV"
+    topology_present = False
+
+    item_name_to_id = item_table
+    location_name_to_id = location_table
+
+    data_version = 1
+    forced_auto_forfeit = False
+
+    options = v6_options
+
+    name_items = {name for name, id in item_table.items()}
```

Comment:
this list is already provided as "item_names" on the World object, generated from item_name_to_id.